onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -399,6 +411,55 @@ def _get_quantized_weight(self, initializer, qType):
 
         return weight
 
+    def _get_quantized_weight_per_channel(self, initializer, qType, channel_axis, symmetric=False, reduce_range=False):
```

Comment:
>reduce_range [](start = 99, length = 12)

do you need reduce_range here as we already have one  self.reduce_range?

Reply:
This is the same case as above. I implemented it to enable `reduce_range` only if both `self.reduce_range` and `reduce_range` are true. This is for controlling operator-wide behaviors. In case of BERT, we want to have full range for embedding matrix and 7-bit for the others.